docs/docusaurus.config.ts

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-
+require('dotenv').config();
```

Comment:
Does this work? We compile the site using ESM, so I wouldn't expect a `require` to work. We should use `import` syntax.

Reply:
Also, is it needed? I believe we already invoke dotenv as part of the build.